src/envoy/tools.clj

```diff
@@ -33,13 +39,18 @@
     (re-matches #"^(true|false)$" v) (Boolean/parseBoolean v)
     (re-matches #"\w+" v) v
     :else
-    (try 
-      (let [parsed (edn/read-string v)]
-        (if (symbol? parsed)
-          v
-          parsed))
+    (try
+      ;;default json no symbol
+      (json/parse-string-strict v true)
          (catch Throwable _
-           v))))
+         ;;fallback to edn for compatibility
+         (try
+           (let [parsed (edn/read-string v true)]
```

Comment:
this does not work: throws, since `edn/read-string` does not take a boolean as its second argument